<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Ash27" failures="0" tests="5" name="com.fortrea.xmr.tests.LoginTest" time="41.821" errors="1" timestamp="2025-08-06T01:14:58 IST" skipped="0">
  <testcase classname="com.fortrea.xmr.tests.LoginTest" name="loginWithValidCredentials" time="11.492"/>
  <testcase classname="com.fortrea.xmr.tests.LoginTest" name="loginWithInValidCredentials" time="10.245"/>
  <testcase classname="com.fortrea.xmr.tests.LoginTest" name="TC_Login_004_passwordRequiredValidation" time="9.446"/>
  <testcase classname="com.fortrea.xmr.tests.LoginTest" name="TC_Login_003_blankUsernameOrPassword" time="5.775"/>
  <testcase classname="com.fortrea.xmr.tests.LoginTest" name="TC_Login_006_sessionExpiryCheck" time="4.863">
    <error message="Cannot invoke &quot;org.openqa.selenium.WebDriver.navigate()&quot; because &quot;com.fortrea.xmr.tests.LoginTest.driver&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.WebDriver.navigate()" because "com.fortrea.xmr.tests.LoginTest.driver" is null
at com.fortrea.xmr.tests.LoginTest.TC_Login_006_sessionExpiryCheck(LoginTest.java:67)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
    </error>
  </testcase> <!-- TC_Login_006_sessionExpiryCheck -->
</testsuite> <!-- com.fortrea.xmr.tests.LoginTest -->
